config/channels/in-memory-channel/deployments/dispatcher.yaml

```diff
@@ -63,6 +63,10 @@ spec:
                 fieldPath: metadata.name
           - name: CONTAINER_NAME
             value: dispatcher
+          - name: MAX_IDLE_CONNS
+            value: "1000"
+          - name: MAX_IDLE_CONNS_PER_HOST
+            value: "1000"
```

Comment:
not against this (actually I like this),  but here is one limitation - when using the knative operator:

https://github.com/knative/eventing/issues/4593

Reply:
this is not a stopper for me... but just raising the concern :smile:

Reply:
@lionelvillard @houshengbo with the recent operstor fix... this is covered over there too? 

Or does this need extra tweak in operator repo?

Reply:
Operator does not implement anything new for configuring the env vars.
When new env vars are added into the manifests, they will be applied once the operator runs the reconciling loop.

For more configuration with the env vars, operator does not support it by CRD.